app/helpers/application_helper.rb

```diff
@@ -537,14 +537,12 @@ def about_links
         :name => "about"
       }
     ]
-    if @community_customization && !@community_customization.how_to_use_page_content.blank?
```

Comment:
If I understood correctly, this will now change the behaviour so that marketplaces which do NOT have any how it works content (on purpose?) will now have the link there and it will take them to blank page. How should we handle those cases? I see two options:

1. Let's decide that there's no way to hide the How to use page. The page will be there, always. (Basically that's what this pull request does)
2. Let's add a new database column `how_to_use_page_in_use` and based on the value on that column we can decide whether or not show the link. This would also require some UI work (add a checkbox "Show How To Use page" or something)

I'd say number 1 is good, but then we need to handle somehow those marketplaces that actually on purpose want to hide the how to use page.